jupyter_vscode_proxy/__init__.py

```diff
@@ -1,35 +1,66 @@
+from typing import Callable, List, Dict, Any
+
 import os
 import shutil
 
 
-def setup_vscode():
-    def _get_vscode_cmd(port):
-        executable = "code-server"
+def _get_inner_vscode_cmd() -> List[str]:
+    return [
+        "code-server",
+        "--auth",
+        "none",
+        "--disable-telemetry",
+    ]
+
+
+def _get_inner_openvscode_cmd() -> List[str]:
+    return [
+        "openvscode-server",
+        "--without-connection-token",
+        "--telemetry-level off",
```

Comment:
Shouldn't this be split?
```suggestion
        "--telemetry-level",
        "off",
```